### ISSUE TITLE
project still on ???

### ISSUE THREAD

**Issue by nnako** [fromcouch/refreak#54]
Hi,

I would be interested in open source task management software (and its development). Is this project still ON? Or was it abandoned?

Are there any screenshots to show the current user interface of refreak?

Thanks.

**Comment by fromcouch** [comment]
No, sorry, it's over. We are working in a new one. But not yet.
You can install and tryit if you want. 

Regards,

**Comment by nnako** [comment]
a pitty... :-(

But is there any example still online so that I could have a look at it?

**Comment by fromcouch** [comment]
So sorry, just removed test site few months ago.

**Comment by fromcouch** [comment]
I just remember i have one working copy here:
http://ebavs.refreak.net/
admin/password

**Comment by nnako** [comment]
nice!!!

**why did you stop development? what features were missing? what alternative do you use now?**

some things I recognized at first glance (maybe because of current version):
- values for percentages (status) seem to be fixed to just a few
- values for context ('work',...) seem to be fixed to just a few
- task list does not change according to settings at the top of window (with refresh)
-

**Comment by zjhxmjl** [comment]
good project!why stop development?
